test/LibYear.Core.Tests/ProjectFileManagerTests.cs

```diff
@@ -162,4 +163,42 @@ public async Task CanUpdateProjectFiles()
 		//assert
 		Assert.Contains("test1", updated);
 	}
+
+	[Fact]
+	public async Task GetAllProjectsWithRecurseFlag()
+	{
+		//arrange
+		var fileSystem = new FileSystem();
+		var fileManager = new ProjectFileManager(fileSystem);
+
+		//act
+		var projects = await fileManager.GetAllProjects(["./FileTypes"], true);
```

Comment:
Just `"FileTypes"` should be good here, so we're not including extra *nix path separators when running on Windows -- doesn't fail, just for consistency